gcip/core/rule.py

```diff
@@ -34,10 +63,47 @@ def __init__(
         self._allow_failure = allow_failure
 
     def never(self) -> Rule:
+        """
```

Comment:
Is this method necessary? I think, this could go into a different class or in `addons`? 
In my opinion, this in not a base functionality of the Rule in GitLab CI

Reply:
Absolutely necessary 😆 I created this method on purpose and used it so often. Yes, it doubles existing functionality, but in a very practical way. It is too convenient and thus often to use to removing it here :)

Reply:
That you created the method on purpose is totally clear. But this makes the API of the core modules more like a general purpose API. 
Mh, that makes me a little bit ...

Reply:
Ok, I know what you mean and in other scenarios I would argue the same.

I grepped my whole git directory and saw, that I didn't use `never()` that much.

I remove `never()` and then the obsolete `copy()` from this class.

Reply:
Phew - I recognized that it is super complicated to implement the `never` switch from outside the rule class. Let's look at this in a peer 2 peer session.